DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -142,9 +162,52 @@ + (BOOL)isEmptyString:(NSString *)string
     return string == nil || [string isEqualToString:@""];
 }
 
++ (BOOL)isNotEmptyString:(NSString *)string
+{
+    return ![[self class] isEmptyString:string];
+}
```

Comment:
Would `isNotEmptyString` along with `attributedTextWithString` be useful in a DK category?